zws40/Intro_to_CodeChat.c

```diff
@@ -0,0 +1,140 @@
+// ====================
+// Solving Mars Lander
+// ====================
+
+// This document will walk you through solving the Mars Lander puzzle on `codingame.com <https://www.codingame.com/ide/puzzle/mars-lander-episode-1>`_
+
+// .. image:: mars_lander_ss_png.png
+
+// The puzzle focuses on the use of conditions in the C language
+
```

Comment:
Put `//` comments to connect these lines, to avoid unnecessary space between paragraphs.